core/src/avm2/object/xml_object.rs

```diff
@@ -118,35 +118,34 @@ impl<'gc> TObject<'gc> for XmlObject<'gc> {
                         return Ok(Value::Undefined);
                     }
                 }
+            }
 
-                let matched_children = if let E4XNodeKind::Element {
```

Comment:
Is this just a formatting change? If so, could you revert this file back to the original state?

Reply:
No the indentation changed. local_name is None for any.

Reply:
I see - thanks